pkg/storaged/jobs.js

```diff
@@ -175,12 +175,19 @@ define([
                 return job(a).StartTime - job(b).StartTime;
             }
 
+            function filter_job(path) {
+                var j = job(path);
+
+                var age_ms = new Date().getTime() - (j.StartTime/1000 - client.time_offset);
```

Comment:
I think all the calls for a list of jobs should have the same reference time to calculate age. Calling new Date() so often feels inefficient and I would expect the "time left" to represent one snapshot in time.